### PR TITLE
S3 put reader

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

**Comment by mburst** [comment]
:m: :a: :x:

File: server/server.go [hunk 1]
```diff
@@ -421,7 +421,7 @@ func (s *Server) Configure(muxer *http.ServeMux) {
 		storeFile, err := saveToTmp(storeReader)
 		if err != nil {
 			resp := ServerResponse{
-				Status: http.StatusBadRequest,
+				Status: http.StatusInternalServerError,
 				Error:  "Error saving originial Image!",
```
Comment (lospro7) [inline server/server.go]: Not your change, but just happened to notice `originial`

**Comment by lospro7** [comment]
:+1:

**Comment by pastudan** [comment]
:+1:

**Pull request merged**